### ISSUE TITLE
Raised error when log_cosh loss function used with ragged Tensor

### ISSUE THREAD

**Issue by shMotaharpour** [keras-team/keras#16062]
**System information**.
I used Google Colab
TensorFlow version with:
v2.7.0-0-gc256c071bb2 2.7.0

**Describe the problem**.

To use the function LogCosh in RNN networks as error function, but this function does not accept the ragged tensor.

**Describe the current behavior**.
It raise ValueError  

**Standalone code to reproduce the issue**.

loss = tf.keras.losses.LogCosh(reduction='none', name='log_cosh')
x = tf.ragged.constant([[1,2,3],[1,2]])
loss(x, x)

**Comment by shMotaharpour** [comment]
Yes, it was useful
Thank you so Much

**Comment by sushreebarsa** [comment]
@shMotaharpour Thank you for the quick update!
Could you please move this issue to closed status if it is resolved for you?
Thanks!
